apps/web/src/locales/en_IN/messages.json

```
@@ -2370,7 +2370,7 @@
     "message": "Licence file"
   },
   "licenseFileDesc": {
-    "message": "Your licence file will be named something like $FILE_NAME$",
+    "message": "Your license file will be named something like $FILE_NAME$",
```

🤔 I'm not as certain about how India's spelling works, but I assume that as a former colony of Britain, that they use the British spelling.

```suggestion
    "message": "Your licence file will be named something like $FILE_NAME$",
```